pom.xml

```diff
@@ -25,8 +25,8 @@
         <module>quantifier</module>
         <module>depparse</module>
         <module>prepsrl</module>
-        <!--pipeline should always be the last item to compile, since some of the previous dependencies have direct dependency on the pipeline-->
```

Comment:
I think maven compiles them in their right order, no matter what order you put here.